File: elevator/python/Passenger.py
class Passenger:

    currentFloor = 0
    destinationFloor = 0

    def __init__(self, currrentfloor, destinationfloor):
        self.currentFloor = currrentfloor
        self.destinationFloor = destinationfloor

    def arrived(self):
        return self.currentFloor == self.destinationFloor

    def getCurrentFloor(self):
        return self.currentFloor

    def setCurrentFloor(self, currentfloor):
        self.currentFloor = currentfloor

    def getDestinationFloor(self):
        return self.destinationFloor

    def __str__(self):
        return "Passenger{" + "currentFloor=" + self.currentFloor + ", destinationFloor=" + self.destinationFloor + "}"

File: elevator/java/src/main/java/elevator/Passenger.java
package elevator;

public class Passenger {

    private int currentFloor;

    private final int destinationFloor;

    public Passenger(int currentFloor, int destinationFloor) {
        this.currentFloor = currentFloor;
        this.destinationFloor = destinationFloor;
    }

    public boolean arrived() {
        return currentFloor == destinationFloor;
    }

    public int getCurrentFloor() {
        return currentFloor;
    }

    public void setCurrentFloor(int currentFloor) {
        this.currentFloor = currentFloor;
    }

    public int getDestinationFloor() {
        return destinationFloor;
    }

    @Override
    public String toString() {
        return "Passenger{" +
                "currentFloor=" + currentFloor +
                ", destinationFloor=" + destinationFloor +
                '}';
    }

}
